services/polymart/polymart-raiting.tester.js

```diff
@@ -0,0 +1,27 @@
+import { isStarRating, withRegex } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('Stars - Polymart Plugin (id 323)')
+  .get('/stars/323.json')
+  .expectBadge({
+    label: 'rating',
+    message: isStarRating,
+  })
+
+t.create('Stars - Invalid Resource (id 1)').get('/stars/1.json').expectBadge({
+  label: 'rating',
+  message: 'not found',
+})
+
+t.create('Rating - Polymart Plugin (id 323)')
+  .get('/rating/323.json')
+  .expectBadge({
+    label: 'rating',
+    message: withRegex(/^(\d*\.\d+)(\/5 \()(\d+)(\))$/),
```

Comment:
We've got a variety of service tests all defining slightly different regular expressions to do roughly the same thing e.g:

- https://github.com/badges/shields/blob/b624179b8d7b138c756b98f6c912ae282f7d31c1/services/pkgreview/package-rating.tester.js#L5-L7
- https://github.com/badges/shields/blob/b624179b8d7b138c756b98f6c912ae282f7d31c1/services/spiget/spiget-rating.tester.js#L19
- https://github.com/badges/shields/blob/b624179b8d7b138c756b98f6c912ae282f7d31c1/services/open-vsx/open-vsx-rating.tester.js#L5

which we should pull out into a shared helper in https://github.com/badges/shields/blob/master/services/test-validators.js

I reckon it doesn't have to be a blocker to merging this, but I'm going to at least just tag here https://github.com/badges/shields/issues/4902 to remind us. It might make a nice next PR to to work on if you were looking for a follow-up :)

Reply:
Ok, should I leave it like that or wait until you add it to the shared helper?

Reply:
lets leave it as it is for this PR